### PR TITLE
add limit param for /field_values

### PR DESCRIPTION
**Pull request by Loori-R**
add `limit` param for the `/field_values` request.

#75
image

### CODE REVIEW

File: src/components/QueryEditor/QueryBuilder/components/QueryBuilderFilters/QueryBuilderFieldFilter.tsx [hunk 1]
```diff
@@ -68,7 +68,7 @@ const QueryBuilderFieldFilter = ({ datasource, filter, query, indexPath, timeRan
     const setterValues = type === FilterFieldType.Name ? setFieldNames : setFieldValues
 
     setterLoading(true)
-    const list = await datasource.languageProvider?.getFieldList({ type, timeRange, field, })
+    const list = await datasource.languageProvider?.getFieldList({ type, timeRange, field, limit: 999 })
```
Comment (dmitryk-dk) [inline src/components/QueryEditor/QueryBuilder/components/QueryBuilderFilters/QueryBuilderFieldFilter.tsx:71]: Can we use const for the limit value?

**Review by dmitryk-dk — DISMISSED** [review]
LGTM! Added one comment

**Review by hagen1778 — APPROVED** [review]

File: src/components/QueryEditor/QueryBuilder/components/QueryBuilderFilters/QueryBuilderFieldFilter.tsx (hunk 1 above)
Comment (Loori-R) [reply src/components/QueryEditor/QueryBuilder/components/QueryBuilderFilters/QueryBuilderFieldFilter.tsx:71]: I thought we could add a configuration field for this limit in the data source settings—similar to what's done here: https://github.com/VictoriaMetrics/victoriametrics-datasource/pull/91. I'll update the PR.

Comment (dmitryk-dk) [reply src/components/QueryEditor/QueryBuilder/components/QueryBuilderFilters/QueryBuilderFieldFilter.tsx:71]: Yeah, I think it is better than hard code the value

Comment (Loori-R) [reply src/components/QueryEditor/QueryBuilder/components/QueryBuilderFilters/QueryBuilderFieldFilter.tsx]: updated

**Review by dmitryk-dk — DISMISSED** [review]
LGTM!

**Review by dmitryk-dk — APPROVED** [review]
LGTM!

**Pull request merged**